### ISSUE TITLE
Node http client

### ISSUE THREAD

**Issue by edongashi** [jmazouri/SharpPad#1]
Hi, thank you for this great tool.

Being able to use the same output from multiple environments would be a fantastic addition.

Considering the .net client communicates via HTTP, how difficult would it be to implement a node client for this protocol?

Some ideas:
```js
import dump from 'SharpPad'

await dump(myObject);
myObject.dump() // maybe add it to Object.prototype?
```

**Comment by EdonGashi** [comment]
I managed to hack around and generated the JSON to fit with JavaScript types:

```js
[1, 2, 3].dump();
'123'.dump();
(123).dump();
true.dump();
new Date().dump();
/123/.dump();
Symbol('123').dump();
({ a: 1, b: 2, c: 3 }).dump();
[{ a: 1, b: 3, c: { d: 4, f: 6 } }, { a: 3, b: 4, c: 2 }, { a: 2, b: [1, 2, 3], c: 3 }].dump();

class Person {
  constructor(name, age) {
    this.name = name
    this.age = age
  }
}

new Person('Test', 25).dump()
```

Looking :fire:

![image](https://user-images.githubusercontent.com/12145268/36627641-f5115cb8-1945-11e8-984f-f691f2c5c063.png)

I will try cleaning up and I can PR where we can discuss further.

**Comment by jmazouri** [comment]
Thank you for the PR - I have merged it (#2). Your work with the Javascript client is very interesting, I hope to provide more client implementations and a more generic server protocol soon.
